tests/unit/components/inputs/autocomplete-test.js

```diff
@@ -116,4 +116,54 @@ describe('Unit: frost-bunsen-input-autocomplete', function () {
       expect(result).to.equal(undefined)
     })
   })
+
+  describe('clear filter when value cleared', function () {
+    beforeEach(function () {
+      component.setProperties({
+        filter: 'foo',
+        value: 'boop',
+        _isTyping: false,
+        _focused: false
+      })
+      sinon.spy(component, 'set')
+    })
+    afterEach(function () {
+      component.set.restore()
```

Comment:
Is this required? I thought it's a blank state on the next test.

Reply:
I guess not, since the component is re-created each test